nova/virt/vmwareapi/driver.py

```diff
@@ -533,6 +533,10 @@ def get_inventory(self, nodename):
                 }
         return result
 
+    def prepare_for_spawn(self, instance):
+        """Prepare to spawn instance."""
```

Comment:
This comment doesn't give any information the name of the function doesn't tell already. We could say, we do pre-checks here or just leave it out.

Reply:
Fixed